src/registry.rs

```diff
@@ -3552,7 +3552,7 @@ impl CheckKind {
             CheckKind::FailWithoutMessage => "No message passed to `pytest.fail()`".to_string(),
             CheckKind::AssertInExcept(name) => {
                 format!(
-                    "Found assertion on exception {name} in except block, use pytest.raises() \
+                    "Found assertion on exception {name} in except block, `use pytest.raises()` \
```

Comment:
I think the backtick should come after `use` here

Reply:
Thanks for the catch!